src/plugins/intel_cpu/src/cpu_memory.cpp

```diff
@@ -219,17 +205,26 @@ void MemoryMngrWithReuse::setExtBuff(void *ptr, size_t size) {
     m_data = decltype(m_data)(ptr, release);
 }
 
-bool MemoryMngrWithReuse::resize(size_t size) {
+bool MemoryMngrWithReuse::resize(size_t size, const ov::element::Type& type) {
     constexpr int cacheLineSize = 64;
     bool sizeChanged = false;
     if (size > m_memUpperBound) {
-        void *ptr = dnnl::impl::malloc(size, cacheLineSize);
+        void* ptr = nullptr;
+        if (type == ov::element::string) {
+            ptr = new OvString[size / ov::element::string.size()];
```

Comment:
should we round up the expression `size / ov::element::string.size()` ?

Reply:
Fixed by different way